app/views/foods/shoping_list.html.erb

```diff
@@ -0,0 +1,31 @@
+<h1 class='text-center mt-5 mb-5'>Shopping List</h1>
+
+<div class="d-flex justify-content-around mx-auto mb-5" style="width: 80%">
+  <h4>Total items: <%= @shoping_list.length %></h4>
+  <% cost = 0
+    @shoping_list.each do |food| 
+    cost += (Food.find_by(name: food[0]).price) * food[1]
+  %>
+  <% end %>
+  <h4>Total cost: $<%= cost %></h4>
+</div>
+
+<table class="table mx-auto text-center" style="width: 80%">
+  <thead>
+    <tr>
+      <th scope="col">Product</th>
+      <th scope="col">Quantity</th>
+      <th scope="col">Price</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @shoping_list.each do |food| %>
+      <tr> 
+        <td><%= food[0] %></td>
+        <td><%= food[1] %></td>
+        <td>$<%= (Food.find_by(name: food[0]).price) * food[1] %></td>
+
+      </tr>
+    <% end %>
+  </tbody>
+</table>
```

Comment:
We believe this is working correctly. The shopping list displays all the ingredient that the current user needs to be able to make all the PUBLIC RECIPIES.

- As mentioned in the requirements: 

> - General shopping list view (only for a group that has less than 3 members):
>   - Should show the list of food that is missing for all recipes of the logged-in user (compare the list of food for all recipes with the general food list of that user).
>   - Should count the total food items and total price of the missing food.

- The comparison is made with **all the recipes(Public recipes)** and the current food that the user has. Which is done correctly. 

- If the user has more food than needed it will not be displayed in the shopping list, as well as if a recipe is not public it will not count towards the food requirements.
![image](https://user-images.githubusercontent.com/114885671/233749915-abad8ff7-3e1f-4818-9ac3-d0e9cbb41c4e.png)

Please explain exactly what needs to be done because we believe this is working correctly @tresorsawasawa